ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/BeaconStateUtil.java

```diff
@@ -116,6 +114,7 @@ public static BeaconStateWithCache get_genesis_beacon_state(
    */
   public static ArrayList<CrosslinkCommittee> get_crosslink_committees_at_slot(
       BeaconState state, long slot, boolean registry_change) throws IllegalArgumentException {
+    //     epoch: 3005 current_epoch: 3005 previous_epoch: 67108864 next_epoch: 3006
```

Comment:
Can u remove these comments if they are not needed?

Reply:
Whoops thanks for catching!